src/main/java/com/libraryapp/BookCatalogue.java

```diff
@@ -0,0 +1,65 @@
+package com.libraryapp;
+
+import jakarta.persistence.*;
+
+@Entity
+public class BookCatalogue {
+    @Id
+    @SequenceGenerator(
+            name = "book_id_sequence",
+            sequenceName = "book_id_sequence"
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "book_id_sequence"
+    )
+    private Integer bookID;
+    private String author;
+    private String title;
+    private Boolean availability;
```

Comment:
do czego jest to pole?

Reply:
To są zmienne do sequence generatora, dzięki którym z automatu tworzy tabelę book o danych atrybutach i kluczu bookId, potem sam wysyła wszystkie zapytania sql przez kontrolery